src/profile/v100/profile.rs

```diff
@@ -525,6 +535,77 @@ mod tests {
         );
     }
 
+    #[test]
+    fn check_if_profile_json_contains_legacy_p2p_links_when_p2p_links_are_present(
+    ) {
+        let value = r#"
+        {
+            "appPreferences": {
+              "p2pLinks": [
+                {
+                  "connectionPassword": "babebabebabebabebabebabebabebabebabebabebabebabebabebabebabebabe",
+                  "displayName": "Brave on PC"
+                }
+              ]
+            }
+          }
+        "#;
+        assert_eq!(
+            true,
+            SUT::check_if_profile_json_contains_legacy_p2p_links(
+                value.as_bytes()
+            )
+        );
+    }
+
+    #[test]
+    fn check_if_profile_json_contains_legacy_p2p_links_when_empty_json() {
+        let value = r#"
+        {
+            
+          }
+        "#;
+        assert_eq!(
+            false,
+            SUT::check_if_profile_json_contains_legacy_p2p_links(
+                value.as_bytes()
```

Comment:
I think this can be `SUT::check_if_profile_json_contains_legacy_p2p_links(BagOfBytes::new())`